src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -3,22 +3,20 @@ class SendEventEmailsJob < ApplicationJob
 
   def perform
     Event::Base.where(mails_sent: false).order(created_at: :asc).limit(1000).each do |event|
-      subscribers = event.subscribers
+      email_subscribers = event.subscribers_for_channel(:instant_email)
+      event.update_attributes(mails_sent: true) if email_subscribers.empty?
 
-      if subscribers.empty?
-        event.update_attributes(mails_sent: true)
-        next
-      end
-
-      NotificationCreator.new(event).call
-      send_email(subscribers, event)
+      NotificationCreator.new(event, :web).call
```

Comment:
Why are we calling the notification creation for `:web` and `:rss` here? I don't understand :thinking:

Reply:
This PR is the second part of #9405, So the user will have different channels for the same `subscription`. The `NotificationCreator` will create the notification if the user has a subscription for the channel `rss` or `web`.

Reply:
Okay